three/index.d.ts

```diff
@@ -261,7 +261,7 @@ declare namespace THREE {
         reset(): AnimationAction;
         isRunning(): boolean;
         startAt(time: number): AnimationAction;
-        setLoop(mode: boolean, repetitions: number): AnimationAction;
+        setLoop(mode: AnimationActionLoopStyles, repetitions?: number): AnimationAction;
```

Comment:
The code itself doesn't seem to imply that `repetitions` is optional. Do you have anything that indicates otherwise?